tests/test-jpms/test-websocket-core/src/main/java/module-info.java

```diff
@@ -0,0 +1,20 @@
+//
+// ========================================================================
+// Copyright (c) 1995-2021 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+module org.eclipse.jetty.websocket.core.tests
```

Comment:
Please add javadoc explaining that this module descriptor is there so that the tests are run in JPMS mode, therefore testing the JPMS module descriptors of the dependencies involved.